vterm/ops.go

```diff
@@ -186,12 +186,12 @@ func (v *VTerm) putChar(ch rune, wide bool) {
 		Style:  v.Cursor.Style,
 	}
 
-	if v.Cursor.Y >= len(v.Screen) {
+	for v.Cursor.Y >= len(v.Screen) {
```

Comment:
Is this an infinite loop? I don't see an exit condition.

Reply:
Oh yeah, agreed, that's not a good solution. I just pushed something that works similar to xDiff